chain/x/zoracle/handler_test.go

```diff
@@ -594,7 +594,7 @@ func TestStopResolveWhenOutOfGas(t *testing.T) {
 		require.Equal(t, types.Success, actualRequest.ResolveStatus)
 	}
 
-	for i := int64(10); i <= 10; i++ {
+	for i := types.RequestID(10); i <= 10; i++ {
```

Comment:
What do you prefer? 

1. `for i := types.RequestID(1); i <= types.RequestID(10); i++ {...}` or 
2. `for i := types.RequestID(1); i <= 10; i++ {...}`

Please make it consistent with other test files.